packages/combobox/src/Combobox.styles.ts

```diff
@@ -9,7 +9,11 @@ import { fontFamilies } from '@leafygreen-ui/tokens';
 import { isArray } from 'lodash';
 import { ComboboxSize, Overflow, State } from './Combobox.types';
 
-const minWidth = 24;
+/**
+ * Width of the widest character (in px)
+ */
+const charWidth = 13; // TODO: update this for Size prop
```

Comment:
How did you get this number?

Reply:
🪄  Magic! jk, just measured the width of the em-dash character